modules/dreamview/proto/simulation_world.proto

```diff
@@ -152,6 +153,12 @@ message Object {
   // perception confidence level. Range: [0,1]
   optional double confidence = 31 [default = 1];
   optional apollo.prediction.ObstaclePriority obstacle_priority = 33;
+
+  //map for sensor measurement
+  map<string, Object> sensors= 36;
```

Comment:
This should be created in message SimulationWorld instead of Object. One object == One Obstacle